### DISCUSSION TITLE
Intestinal permeability (transcellular) formula error in MoBi

### DISCUSSION THREAD

**Discussion by Sarang-Mishra** [Open-Systems-Pharmacology/Forum#1620] [Questions & Problems]
Hi All,
I am facing error when I transferred a working pk-sim model to MoBi. I states that "cannot evaluate formula for Intestinal permeability (trascellular error), earlier I had not faced any error while migrating model from pk-sim to mobi. Does anyone have know, how to resolve it?

Thanks 
![trascellular error](https://github.com/Open-Systems-Pharmacology/Forum/assets/63973987/acfc4be7-06e4-4547-9c0e-c9de1b1bcb74)

**Comment by PavelBal** [comment]
Could you share the PK-Sim project that we can reproduce the issue?
